src/main/frontend/components/FilmTile.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const FilmTile = (props) => {
+  const { id, title, imgUrl, year } = props.film
+
+  return (
+    <div>
+      <a href={`/films/${id}`}>
+        <img src={imgUrl} />
+        <h3>{title} - {year}</h3>
+      </a>
+    </div>
+  )
+}
+
+export default FilmTile
```

Comment:
It actually looks like `Films` don't exist yet, so I would put this on a `Add films to Genre Show` PR